src/js/containers/router/RouterRoutes.jsx

```diff
@@ -120,17 +120,19 @@ const routes = {
             }
         },
         {
-            path: '/bulk_download',
-            parent: '/bulk_download',
+            path: '/download_center',
+            parent: '/download_center',
+            silentlyUpdate: true,
```

Comment:
Should this be `silentlyUpdate`? It seems to me that you would want the pages to reload completely when the URL changes between types.